Crashlytics/Crashlytics/Public/FIRStackFrame.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * The Firebase Crashlytics Stack Frame provides a way to construct the lines of
+ * a stack trace for reporting along with a recorded Exception Model.
```

Comment:
Replace with:

"The Firebase Crashlytics stack frame API provides a way to construct the lines of a stack trace when reporting exception models."

Get rid of "recorded" (bc it's implicit), make "Stack Frame" a non-proper noun (for consistency), remove "along" (implies reporting more than the exception)